package.json

```diff
@@ -50,6 +51,20 @@
     "clean": "rm -rf ./build/ ./dist/ .parcel-cache/",
     "prod:build": "parcel build --public-url ./ ."
   },
+  "jest": {
+    "verbose": true,
+    "transform": {
+      "^.+\\.(ts|tsx)?$": "ts-jest",
+      "^.+\\.(js|jsx)$": "babel-jest"
+    },
+    "modulePaths": [
+      "./src/views/hooks",
+      "./node_modules/"
+    ],
+    "moduleDirectories": [
+      "./node_modules/"
```

Comment:
@Romulus10 This seems like it should resolve the error with looking for ipfs-core